### PR TITLE
Hangman Game 

### PR DESCRIPTION
**Pull request by Ayush7614**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #(issue_no)

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [ ] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [ ] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by Ayush7614** [comment]
@avinashkranjan plss review it this pull is for issue no #278

**Comment by Ayush7614** [comment]
@avinashkranjan plss review it

File: Hangman Game/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# The task is to create a hangman game.
```
Comment (atarax665) [inline Hangman Game/README.md:1]: Readme should strictly follow [this](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md) template.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Rename the images with proper name and remove the PDF file as it isn't needed, screenshots are enough.

**Comment by Ayush7614** [comment]
@avinashkranjan and @atarax665 plss review it

File: Hangman Game/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Package/Script Name
```
Comment (atarax665) [inline Hangman Game/README.md:1]: Your README should look something like [this](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Reddit-Meme-Scraper/README.md). Please make appropriate changes.

**Review by atarax665 — CHANGES_REQUESTED** [review]
The images have not been renamed yet.

File: Hangman Game/README.md (hunk 2 above)
Comment (Ayush7614) [reply Hangman Game/README.md]: > Your README should look something like [this](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Reddit-Meme-Scraper/README.md). Please make appropriate changes.

okk @atarax665 now you see and plss review it and merged it

**Comment by Ayush7614** [comment]
@avinashkranjan sir plss review it

**Comment by Ayush7614** [comment]
@avinashkranjan sir plsss review it

File: Hangman Game/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Create a List of secret words and randomly pick a word. Now represent each word as _ and give user chances to guess the word if the user guesses the word right then replace _ with the word.
+ 
+## Setup Instruction
+
+first install time package in your system open your terminal and write `pip install time` this will install time module in your system and random module is already install in python by itself so don't waste your time. Now make file `program.py` or whatever your choice and write program from program.py file and run it.
+
+Make sure you have installed the **necessary packages** listed in **`requirements.txt`**, then simply run **`program.py`**.
+
+# Screenshots
+[Output](https://github.com/Ayush7614/Amazing-Python-Scripts/blob/master/Hangman%20Game/Output.pdf)
```
Comment (atarax665) [inline Hangman Game/README.md:12]: There is no output.pdf available now. Please add image link here instead.

**Review by atarax665 — COMMENTED** [review]
Rename the images downloaded from whatsapp.

**Comment by Ayush7614** [comment]
now @atarax665 sir plss review it and merged it so i can work on some more issues

**Comment by Ayush7614** [comment]
This pull is for the issue no #245

**Comment by Ayush7614** [comment]
@atarax665 plss review it and merged it

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**